app/src/main/java/com/simplemobiletools/draw/Constants.java

```diff
@@ -9,4 +9,7 @@
     public static final String IS_FIRST_RUN = "is_first_run";
     public static final String IS_DARK_THEME = "is_dark_theme";
     public static final String IS_STROKE_WIDTH_BAR_ENABLED = "is_stroke_width_bar_enabled";
+    // canvas
+    public static final int PREVIEW_DOT_STROKE_SIZE = 5;
+    public static final int PREVIEW_DOT_OFFSET_Y = 120;
```

Comment:
one more thing, add these values in dimens.xml in some dp values, so that they scale properly depending on the mobile resolution